resources/js/components/Litter/AddTags.vue

```diff
@@ -197,6 +209,11 @@ export default {
         };
     },
     computed: {
+        
+        //whether the user wants to show previous tags. return type: boolean
+        show_previous_tags(){
+            return this.user.previous_tags == 1;
```

Comment:
When the value of `previous_tags` is a boolean or number, we can return that if it is `1`. It helps to wrap these statements in brackets too to show its computing something. Can you update the comment block to match the others too?

```
/**
 * Return true to show previous tags
 */
show_previous_tags () 
{
     return (this.user.previous_tags);
}
```